src/pages/comments/Comments.jsx

```diff
@@ -1,9 +1,45 @@
+import moment from "moment";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { fetchCommentsByArticleId } from "../../api";
+
 const Comments = () => {
-    return ( 
-        <div className="comments">
-            Comments
-        </div>
-     );
-}
- 
-export default Comments;
\ No newline at end of file
+  const { article_id } = useParams();
+  const [isLoading, setIsLoading] = useState(true);
+  const [comments, setComments] = useState();
+
+  useEffect(() => {
+    fetchCommentsByArticleId(article_id).then((comments) => {
+      setIsLoading(false);
+      setComments(comments);
+    });
+  }, [article_id]);
+
+  if (isLoading) {
+    return <div className="loader"></div>;
+  }
+
+  //   Posted At: {moment(article.created_at).format("H:mmA")} on{" "}
+  //   {moment(article.created_at).format("DD MMM YY")}
+
+  return (
```

Comment:
There are an awful lot of <div>s in this component - make sure you're using semantic html tags. 
This is a great refresher [on semantic elements](https://www.w3schools.com/html/html5_semantic_elements.asp)